nebula/ui/components/VPNFeatureTour.qml

```diff
@@ -278,46 +145,36 @@ ColumnLayout {
         VPNButton {
             id: resumeButton
 
+            objectName: "primaryButton"
```

Comment:
We should probably make this `objectName` something more specific since the Inspector and some of the tutorial infra rely on being able to find elements by a unique `objectName`.